ui/components/app/loading-network-screen/loading-network-screen.container.js

```diff
@@ -67,7 +68,29 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
+const mergeProps = (stateProps, dispatchProps, ownProps) => {
+  const { history, location, ...restOwnProps } = ownProps;
+
+  const onDisplayed = () => {
+    // Redirect to the intended route if available, otherwise DEFAULT_ROUTE
```

Comment:
Might need a little explanation here.

Also no types scares me, are we confident this doesn't break other usages (or is it only used in 1 place)

Reply:
Actually ignore me. These are mergeProps, which will combine this inside the `connect()` component.

It does take in the `history` and `location` props - which is correctly passed in through instances where this is used.

Reply:
History and location are injected through `react-router-dom` `withRouter`. We could potentially use the library hooks instead of prop drilling.